### ISSUE TITLE
no_std support

### ISSUE THREAD

**Issue by vinaychandra** [azriel91/enum_variant_type#2]
The crate doesn't use any non-core code. It would be trivial to add no_std support by making `std` into `core`

**Comment by azriel91** [comment]
Heya @vinaychandra, I've never written a library with `no_std` support, so #3 is my attempt.

Can you test that it works?

```toml
[patch.crates-io]
enum_variant_type = { git = "https://github.com/azriel91/enum_variant_type.git", branch = "feature/2/no-std" }
```

**Comment by vinaychandra** [comment]
Cool. Thanks! In my quick test, this works as expected.

**Comment by azriel91** [comment]
Heya, this is released in [0.2.1](https://crates.io/crates/enum_variant_type/0.2.1)
